cva6/sim/desc_gen-tests.yaml

```diff
@@ -0,0 +1,511 @@
+# Copyright Google LLC
```

Comment:
This file _looks_ like, but is not, a direct copy of a test-program generator description from riscv-dv (e.g. `riscv-dv/yaml/base_testlist.yaml`), so it may confuse new uses of core-v-verif/cva6.  I would recommend changing the filename to something like `cva6_base_testlist.yaml` and add a section to the top of the comment header to make it distinguishable from a testlist in riscv-dv/yaml.  Perhaps something like this:
```
# Copyright 2022 Thales
#
# CVA6-specific testlist.  Written in the same format as riscv-dv testlists.
#
```

Reply:
Ok it's done for the YAML description

Reply:
Hi @MikeOpenHWGroup, I change the file name to cva6_base_testlist.yaml as you recommend, and now in the script (dv-generated-tests.sh) we can use new test description in the cva6_base_testlist.yaml, and also tests in the riscv-dv, simply to give the list_num variable =0, to re-use tests in the base_testlist.yaml in riscv-dv.
Now the cva6_base_testlist.yaml don't contain any test from the base_testlist file in riscv-dv, but we can re-used these tests using the script